ui/appui-react/tsconfig.json

```diff
@@ -3,6 +3,8 @@
   "compilerOptions": {
     "jsx": "react",
     "resolveJsonModule": true,
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
     "types": ["vitest/globals"]
   },
   "include": ["./src/**/*.ts*"]
```

Comment:
While the new additions to this file look good, it is generally advised to have separate tsconfig files for build vs test.

See https://kettanaito.com/blog/one-thing-nobody-explained-to-you-about-typescript

Reply:
There are also other changes you might want to make to the tsconfig, such as using the new JSX transform or adding `verbatimModuleSyntax`.